.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: ~/repo
     docker:
-      - image: circleci/node:latest
+      - image: cimg/node:16.17.0
```

Comment:
does it break with latest node?

Reply:
@obartra yes -- I believe the older version of node-canvas is incompatible with more modern versions of node. We could upgrade the node-canvas dependency but I guess that should be a separate PR.

Reply:
absolutely 👍